faqs/galaxy/reduce_quota_usage.md

```diff
@@ -0,0 +1,14 @@
+---
+title: How can I reduce quota usage while still retaining prior work (data, tools, methods)?
+area: Account
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+* **Download** Datasets as individual files or entire Histories as an archive. Then *purge* them from the public server.
+* **Copy** Datasets or Histories to another Galaxy server, including your own Galaxy. Then purge.
+* **Copy** your most important Datasets into a new/other History (inputs, results), then purge the original full History.
+* Extract a **Workflow** from the History, then purge it. 
+* **Back-up your work**. It is a best practice to download an archive of your FULL original Histories periodically, even those still in use, as a backup.
```

Comment:
Great! That is much eaiser to read. That would be really nice to add links to other FAQs or training that shows how to download, copy, etc.

Reply:
@annefou thanks for the links. I have just added links to other FAQs. Kindly have a look at the updates and give me your feedback.